src/components/Integrations/AvailableIntegrationsList.js

```diff
@@ -15,6 +15,7 @@ export default function AvailableIntegrationsList({ setSelectedIntegration }) {
     .catch(console.error)
   }, [])
 
+  // comment on un-needed extraction
   const buildLogos = () => {
```

Comment:
this form of "pulling out a piece of the render" function tends to be unnecessary. a one line map in your render code is not going to make it a lot less readable, and it's usually nice to be able to see **everything** that gets rendered from this file in one block of code, as opposed to having to scroll back up to find the `buildLogos` function.

my rule of thumb: If i **actually think it's worth splitting out**, it should be it's own component in a separate file.

For this, i'd just put it all in the return:
```jsx
  return (
    <>
      <div onClick={() => setShowIntegrations(!showIntegrations)} className={"pt-4 flex flex-row text-gray-500"}>
        <div className={"pr-2 my-auto text-lg"}  >
        { showIntegrations
          ? <FaAngleUp />
          : <FaAngleDown />
        }
        </div>
        <p className={"text-lg"}>Data Integrations</p>
        <div className={"ml-2 w-12 h-0 my-auto flex-grow border-dashed border-gray-500 border-b"}></div>
      </div>
      <div className={`${showIntegrations ? null : "hidden"} flex flex-row pt-8`}>
        {
          integrationTiles && integrationTiles.map( integration =>  <IntegrationCard key={integration.name} integration={integration} setSelectedIntegration={setSelectedIntegration} />)
        }
      </div>
    </>
  )
```